Sources/TimeZone.php

```diff
@@ -18,8 +18,6 @@
  */
 class TimeZone extends \DateTimeZone
 {
-	use BackwardCompatibility;
-
```

Comment:
Forgot to remove the $backcompat property.

Reply:
Im not even working on those yet, where they have been left is a marker for project wide search ;)